student.rb

```diff
@@ -1,14 +1,16 @@
-require_relative './person'
+# rubocop: disable Style/OptionalBooleanParameter
```

Comment:
- [OPTIONAL]  `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `rubocop`, it will be nice to fix the issue.

Reply:
Thanks for your suggestion.